grug/vm/rust/src/vm.rs

```diff
@@ -85,6 +85,7 @@ impl Instance for RustInstance {
                     &MockApi,
                     &self.querier,
                 )?;
+
```

Comment:
Remove this extra line. Let's not touch the code that's not the concern of this PR.